core-services/image-mirroring/openshift/mapping_origin_4_6

```diff
@@ -3,9 +3,7 @@ registry.ci.openshift.org/origin/4.6:branding quay.io/openshift/origin-branding:
 registry.ci.openshift.org/origin/4.6:cluster-samples-operator quay.io/openshift/origin-cluster-samples-operator:4.6 quay.io/openshift/origin-cluster-samples-operator:4.6.0
 registry.ci.openshift.org/origin/4.6:cluster-update-keys quay.io/openshift/origin-cluster-update-keys:4.6 quay.io/openshift/origin-cluster-update-keys:4.6.0
 registry.ci.openshift.org/origin/4.6:installer quay.io/openshift/origin-installer:4.6 quay.io/openshift/origin-installer:4.6.0
-registry.ci.openshift.org/origin/4.6:installer-artifacts quay.io/openshift/origin-installer-artifacts:4.6 quay.io/openshift/origin-installer-artifacts:4.6.0
```

Comment:
I'm assuming you didn't actually mean to touch 4.6 given you skipped over 4.15 - 4.7?

Reply:
This was done automatically by `make openshift-image-mirror-mappings` I had no control over it